sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -57,9 +62,50 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
 
       tags ++ branches
     },
+    tlCiReleaseStepSummaryTableInfo := {
+      val map: Map[String, String] = Map("Release version" -> (ThisBuild / version).value)
+      (ThisBuild / apiURL).value.map(_.toString).fold(map)(r => map + ("Api URL" -> r))
+    },
+    tlCiAddResolverInfoToSummary := true,
     githubWorkflowTargetTags += "v*",
     githubWorkflowPublish := Seq(
       WorkflowStep.Sbt(List("tlRelease"), name = Some("Publish"))
     )
   )
+
+  private def renderSummaryTable(results: Map[String, String]): String =
+    results
+      .toList
+      .map { case (k, v) => s"| ${k} | ${v} |" }
+      .mkString(s"| Build Results | |\n| -: | :- |\n", "\n", "\n")
+
+  override def projectSettings: Seq[Setting[_]] = Seq(
+    publish := {
+      val table: Map[String, String] = tlCiReleaseStepSummaryTableInfo.value
+      val maybeMavenResolverUrl: Option[(String, String)] =
+        (ThisBuild / publishTo).value.collect {
+          case x: MavenRepo => (x.name, x.root)
+          case x: MavenRepository => (x.name, x.root)
+        }
```

Comment:
@armanbilge I tried this impl to get the resolver URL just in the cases the resolver is a maven repo with a URL, but it should be definitively tested

Reply:
For the moment these settings are producing a bunch of 👇 

### sbt-typelevel-github-actions Publication Summary
| Build Result | Value |
| -: | :- |
| Release version | 0.4.18-24-5fe499e-20230205T100148Z-SNAPSHOT |
| Api URL | https://oss.sonatype.org/service/local/repositories/snapshots/archive/org/typelevel/sbt-typelevel-docs_2.12/0.4.18-24-5fe499e-20230205T100148Z-SNAPSHOT/sbt-typelevel-docs_2.12-0.4.18-24-5fe499e-20230205T100148Z-SNAPSHOT-javadoc.jar/!/index.html |
| Scala Versions | 2.12.17 |

Reply:
> > as with `publishLocal` the publishTo doesn't get populated
> 
> I mean, that's fine, since running `publishLocal` in CI doesn't put it anywhere you can use it anyway :)

Not sure I follow. What I meant is that "on my laptop" the publishing fails before the command completes, and the only thing I had the chance to test is `publishLocal` overridden in the same way this `publish` is. To REALLY test with `publish` we should... well... publish :D


> I am also pretty sure it won't work on tagged releases which go to maven central. But that's also fine because everybody knows where to find those.

Hmm, where's the configuration for tagged releases? I might take a look to try to make it compatible.

Reply:
> Hmm, where's the configuration for tagged releases? I might take a look to try to make it compatible.

Somewhere in https://github.com/xerial/sbt-sonatype

The reason I suspect it won't work is because sbt-sonatype does a "bundle release" when deploying to maven. So instead of setting `publishTo` to a sonatype resolver, IIUC it sets `publishTo` to your local file system, and then atomically uploads that as a bundle to Sonatype.

Anyway, I wouldn't worry about it. We can land this, and futz with that in a follow-up :)

Reply:
> Anyway, I wouldn't worry about it. We can land this, and futz with that in a follow-up :)

Sounds good, we can handle these improvements atomically. Also, I saw that there are a bunch of enhancement proposals for this plugin that might deserve some attention now that the shade curtain between me and sbt plugins is getting thinner :D

I don't think I have nothing else to develop in this PR, I'll wait for the definitive review.